### PR TITLE
[FIX][base_address_city]Add in the domain city_id for only show the c…

### PR DESCRIPTION
**Pull request by IsabelEsparza**
…ities of the selected state

Description of the issue/feature this PR addresses:
Selection cities shows all cities and not only from the state selected.

Current behavior before PR:
Selection cities shows all cities.

Desired behavior after PR is merged:
Selection cities shows only cities from state selected.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by pedrobaeza** [comment]
This avoids to select cities with country, but without state.

File: addons/base_address_city/models/res_partner.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def _fields_view_get_address(self, arch):
                               'default_name': city,
                               'default_zipcode': zip,
                               'default_state_id': state_id}"
-                    domain="[('country_id', '=', country_id)]"
+                    domain="[('country_id', '=', country_id),('state_id', '=', state_id)]"
```
Comment (mreficent) [inline addons/base_address_city/models/res_partner.py]: `'state_id', 'in', [state_id, False]`

**Comment by mart-e** [comment]
Hi,

I agree on the report but this doesn't seem to work, does it?
If I have 
- city 1 - state A - country X
- city 2 - state B - country X
- city 3 - state C - country Y

When selecting country X and state A, the domain will be "country = X OR state in (A, False)", so both city 1 and city 2 will match.
Shouldn't it be an AND clause instead of an OR ?

**Comment by C3POdoo** [comment]
Dear @IsabelEsparza,

Thank you for your contribution but the version 12.0 is no longer supported.
We only support the last 3 stable versions so no longer accepts patches into this branch.

We apology if we could not look at your request in time.
If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

_This is an automated message._

**Pull request closed without merging**